SW.Tools/Entities/Comprobante.cs

```diff
@@ -1170,6 +1172,21 @@ public bool DescuentoSpecified
                 this.descuentoFieldSpecified = value;
             }
         }
+
+        [System.Xml.Serialization.XmlIgnoreAttribute()]
+        public bool NoIdentificationSpecified
+        {
+            get
+            {
+                return this.noIdentificationFieldSpecified;
```

Comment:
puedes hacer esto y es mas eficiente y siempre te regresaria el valor true/false ademas de que ya no lo tendrias que validar en tus otros ifs que tienes abajo
`return !string.IsNullOrEmpty(this.noIdentificacion)`